tests/Shared/WorkloadTesting/data/nuget8.config

```diff
@@ -8,6 +8,7 @@
     <clear />
     <add key="built-local" value="%BUILT_NUGETS_PATH%" />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
```

Comment:
I don't recall why I had to that one :( I will remove this in the follow up PR, and see what fails.
Also, with using a single nuget.org for sources this will coming from the root nuget.config .